File: web_fund/algos/algo25.js
// 25. Always Hungry - Create a function that accepts an array, and prints
// "yummy" each time one of the values is equal to "food".  If no array
// values are "food", then print "I'm hungry" once

function alwaysHungry(arr) {
    count = 0;
    for (var i = 0; i < arr.length; i++) {
        if (arr[i] == "food") {
            console.log("yummy");
            count++
        }
    }
    if (count == 0) {
        console.log("I'm hungry")
    }
}

alwaysHungry([1, 3, "food", 5, "food", "cat", 2, "food"])
alwaysHungry([1, 3, -9, "cat", 0, "dog"])

File: practice_algos/python/algo25.py
# 25. Always Hungry - Create a function that accepts an array, and prints
# "yummy" each time one of the values is equal to "food".  If no array
# values are "food", then print "I'm hungry" once

def always_hungry(arr):
    count = 0
    for item in arr:
        if item == "food":
            print("yummy")
            count += 1
    if count == 0:
        print("I'm hungry")
    return


always_hungry(["food", 1, 2, "food", 3, "food"])
always_hungry([1, 2, 3])
